released/packages/coq-geocoq-pof/coq-geocoq-pof.2.5.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer:   "Pierre Boutry <contact@pierre.boutry.fr>"
+
+homepage:     "http://geocoq.github.io/GeoCoq/"
+dev-repo:     "git+https://github.com/GeoCoq/GeoCoq.git"
+bug-reports:  "https://github.com/GeoCoq/GeoCoq/issues"
+license:      "LGPL-3.0-only"
+
+synopsis:     "A formalization of foundations of geometry in Coq"
+description:  "This is a virtual package that can be safely removed."
+
+depends:  ["coq-geocoq-algebraic" { = version }]
+
+tags: [
+  "category:Mathematics/Geometry/General"
+  "keyword:geometry"
+  "keyword:Euclidean geometry"
+  "keyword:hyperbolic geometry"
+  "keyword:foundations"
+  "keyword:Tarski"
+  "keyword:parallel postulates"
+  "keyword:model"
+  "keyword:counter-model"
+  "keyword:Cartesian space"
+  "date:2024-03-24"
+]
+authors: [
+  "Pierre Boutry       <contact@pierre.boutry.fr>"
+  "Cyril Cohen         <cyril.cohen@inria.fr>"
+  "Stéphane Kastenbaum <stephane.kastenbaum@gmail.com>"
+]
```

Comment:
@Boutry wouldn't it be better to just replace `coq-geocoq-pof` with `coq-geocoq-algebraic` entirely? I can understand the top-level `coq-geocoq` virtual package, but this one might just cause confusion.

Reply:
Do you mean renaming the older ones as part of the PR to avoid the virtual package? By the way your comment makes me realize that there is a line missing in `coq-geocoq`. It should depend on `coq-geocoq-elements` too so that it installs all the packages.

Reply:
OK, to be very specific I mean it may make sense to remove the file `released/packages/coq-geocoq-pof/coq-geocoq-pof.2.5.0/opam` entirely from this PR, and make `coq-geocoq-algebraic` conflict with *any* version of `coq-geocoq-pof`.

Reply:
Won't it mean that people who have `coq-geocoq-pof.2.4.1` but not `coq-geocoq.2.4.1` will not get offered to upgrade to the `coq-geocoq-pof.2.5.0` which will install `geocoq-algebraic.2.5.0` letting them know of the replacement?

Reply:
But are there really that many people who explicitly depend on `coq-geocoq-pof`? Do you plan to keep a "virtual" `coq-geocoq-pof` package around forever? Otherwise, the problem with "lack of upgrade" will just happen one version further down the road. It's a good idea to only keep at most a top-level virtual package due to overhead of packages.

Reply:
I do not know who uses it so... But yes maybe, people will be made aware of the renaming when they will see that there is a new release but they cannot upgrade to it.

Reply:
OK, fine by me, if you add the missing dependency to the `coq-geocoq` virtual package, I can merge this.